Python/Product/Django/Django.csproj

```diff
@@ -113,6 +113,31 @@
     </Reference>
     <Reference Include="WindowsBase" />
   </ItemGroup>
+  <Choose>

```

Comment:
I considered that, but fairly sure VS won't handle it properly. The C# project system doesn't respect all of MSBuild's functionality...